packages/Babylonian-UI.package/BPProbeMorph.class/instance/newLabelMorph.st

```diff
@@ -0,0 +1,31 @@
+private - ui
+newLabelMorph
+	
+	| container labelSource labelFieldMorph |
+	container := Morph new
+		color: Color transparent;
+		changeTableLayout;
+		listDirection: #topToBottom;
+		vResizing: #shrinkWrap;
+		hResizing: #spaceFill;
+		cellPositioning: #leftCenter;
+		borderWidth: 1;
+		borderColor: Color black;
+		layoutInset: 1;
+		yourself.
+		
+	labelSource := self annotation hasLabel ifTrue: [self annotation label] ifFalse: [''].
+	labelSource := labelSource copyFrom: 2 to: (labelSource size - 1 max: 1). "Removes the quotes brackets"
+	labelFieldMorph := (self 
+		editFieldInitialContent: labelSource
+		onChange: [:str | 
```

Comment:
:aString would be more conform to the coding standards

Reply:
newExpressionMorph also uses :str

Reply:
yes and it's against the guidelines :p